### PR TITLE
Add wiki link to website

### PR DESCRIPTION
**Pull request by LukasGerm**
Added the wiki link to website.

Explain how you tested your changes here:
No testing needed.

Checklist:

- [ ] Tests were added for the new behavior
- [ ] All tests pass (CI will check this if you didn't)
- [ ] Serialized types are in stable-versioned modules
- [ ] Does this close issues? List them:

### CODE REVIEW

**Comment by bkase** [comment]
I didn't think we could fit another link in the nav, but it seems like this does work -- I don't know if we want to use that extra spot for the wiki though. Will share this in the product channel for discussion

**Comment by bkase** [comment]
Echoing Harold's concern here: "[we should] have a discussion before adding another item to the top level nav, especially one that takes users off of the main site."

Maybe you can add the link in the footer for now?

**Comment by LukasGerm** [comment]
Hey @bkase I just moved the link to the footer.

File: frontend/website/src/components/Page.re [hunk 1]
```diff
@@ -76,6 +76,9 @@ module Footer = {
             <Link link="https://github.com/CodaProtocol/coda" name="github">
               {React.string("GitHub")}
             </Link>
+            <Link link="https://codawiki.com" name="wiki">
+              {React.string("Coda Wiki")}
```
Comment (bkase) [inline frontend/website/src/components/Page.re]: Would you mind renaming this to `Community Wiki`?

**Comment by LukasGerm** [comment]
So @bkase I just updated the link name. It should be complete now.

**Comment by Schmavery** [comment]
Looks like you might need to run `yarn reformat` or something (since the build-wallet is failing)

**Comment by bkase** [comment]
Thanks!

**Pull request merged**